### ISSUE TITLE
unlikely and c++20

### ISSUE THREAD

**Issue by hsdk123** [kcat/openal-soft#804]
```
C:\Users\daego\Desktop\Projects\Light.vn-core\core\out\build\x64-Release\_deps\openal-soft-src\al\buffer.cpp(1659): warning C5051: attribute 'unlikely' requires at least '/std:c++20'; ignored
```
Hi, I've recently started getting a bunch of warnings like this - is there a way to evade these?

**Comment by kcat** [comment]
The proper way to handle it would be to complain to Microsoft to not be so pendantic. Unknown/unsupported/unhandled attributes in C++ are not an error, VS knows exactly what that attribute means and what I mean for it to do, and honoring the attribute does not miscompile the code (it's a valid way to optimize branches regardless of any specified attributes). Honoring it still results in a completely valid C++14 binary, and VS it completely capable of optimizing it as indicated, but chooses not to and leaves performance on the table, pessimizing the resulting code.

But I doubt Microsoft would listen to me, and I doubt they'd add an option 'please handle attributes you know about and are required to accept anyway', so I'm going to have to hide the warning. Which also means hiding that it's not optimizing the code as it could/should.

**Comment by kcat** [comment]
Fixed with commit 3f8a3af3637606dae61ba112cb5bd9d2d897e5cb.
